features/api/command/run.feature

```diff
@@ -60,13 +60,13 @@ Feature: Run command
     Then the specs should all pass
 
   Scenario: Non-existing executable
-    Given a file named "bin/cli" does not exist
+    Given a file named "bin/cli-app" does not exist
```

Comment:
Na... The problem is this:

1. The fixture contains a default executable called `bin/cli`
2. We need to rename that file and each test using it, say to `bin/aruba-test-cmd` - this should avoid further conflicts with existing executables
3. I would prefer to stick to `Given a file named "bin/aruba-test-cmd" does not exist`, so we can still use a default command in our fixture and properly set state. We also explicitly say, that we asume, that such a file does not exist.